overlord/snapstate/snapmgr.go

```diff
@@ -209,6 +209,22 @@ func (snapst *SnapState) Block() []snap.Revision {
 
 var ErrNoCurrent = errors.New("snap has no current revision")
 
+// Retrieval functions
+var readInfo = readInfoImpl
+
+func readInfoImpl(name string, si *snap.SideInfo) (*snap.Info, error) {
```

Comment:
readInfoAnyway or similar?  This is not the actual implementation.. it's adding some value on top of it.